effekt/jvm/src/main/scala/effekt/EffektConfig.scala

```diff
@@ -148,6 +148,24 @@ class EffektConfig(args: Seq[String]) extends REPLConfig(args.takeWhile(_ != "--
     group = debugging
   )
 
+  val showDocumentation: ScallopOption[Boolean] = toggle(
+    "show-documentation",
+    descrYes = "Show all documented statements as a JSON",
+    default = Some(false),
+    noshort = true,
+    prefix = "no-",
+    group = debugging
+  )
+
+  val writeDocumentation: ScallopOption[Boolean] = toggle(
+    "write-documentation",
+    descrYes = "Write all documented statements to a JSON in the output directory",
+    default = Some(false),
+    noshort = true,
+    prefix = "no-",
+    group = debugging
+  )
```

Comment:
What do we need the showDocumentation option for?

Reply:
Not sure, it could be used for piping the JSON directly to `jq` or some other program. Mostly it's there because of the similarities to the `showIR` and `writeIR` flags. Should I remove it?

Reply:
We already talked about subcommands at some point

Reply:
(https://github.com/effekt-lang/effekt/issues/534 is the CLI subcommands issue, but the CLI parsing library we use doesn't really support it + it's buggy https://github.com/effekt-lang/effekt/issues/889 + we use the config in a very unstructured and ad-hoc way right now)